LibraBFT/Yasm/Base.agda

```diff
@@ -48,9 +48,11 @@ module LibraBFT.Yasm.Base (ℓ-PeerState : Level) where
       -- Represents an uninitialised PeerState, about which we know nothing whatsoever
       initPS    : PeerState
 
-      -- Initializes a potentially-empty state with an EpochConfig
-      init : PeerId → Genesis → PeerState × List (Action Msg)
+      -- Bootstraps a peer.
+      -- TODO-1 : replace Maybe with Either after adding error type (to mkSysTypeParms ?).
+      bootstrap : PeerId → Genesis → Maybe (PeerState × List (Action Msg))
```

Comment:
I thought we decided to rename `Genesis` to (something like) `BootstrapInfo`, as `Genesis` sounds a bit application-centric and therefore doesn't belong in the system model.  I'm ambivalent regarding `init` vs. `bootstrap` for the function name.

Reply:
One thing at a time.   First step is `Maybe` on `bootstrap`.